modules/dreamview/backend/simulation_world/simulation_world_service.cc

```diff
@@ -942,6 +942,12 @@ void SimulationWorldService::UpdatePlanningData(const PlanningData &data) {
   for (auto &path : data.path()) {
     DownsamplePath(path, planning_data->add_path());
   }
+
+  // Update pull over status
+  if (data.has_pull_over_status()) {
+    planning_data->clear_pull_over_status();
```

Comment:
please move clear_pull_over_status() out of the if statement in case the previous frame has data but current one doesn't.